.git-blame-ignore-revs

```diff
@@ -6,3 +6,4 @@
 # https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html#avoiding-ruining-git-blame
 # Apply noqa markers for all TCH001 violations
 f5e7df1846102d9a62cc9b9110387925ffae60cc
+7bc7d93bd3d8f4c79d192569a6a3d8c81c7d5e17
```

Comment:
Please add a comment explaining above this hash explaining what this change was.

But apart from that I'm not sure this is the kind of change we should ignore in the blame.

Reply:
Works for me.

Reply:
✅

Reply:
It doesn't even apply anymore after Alex made the same change 🤣